Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2144,16 +2146,20 @@ extension BrowserViewController: ContextMenuHelperDelegate {
             actionSheetController.addAction(copyImageLinkAction, accessibilityIdentifier: "linkContextMenu.copyImageLink")
         }
 
-        // If we're showing an arrow popup, set the anchor to the long press location.
-        if let popoverPresentationController = actionSheetController.popoverPresentationController {
-            popoverPresentationController.sourceView = view
-            popoverPresentationController.sourceRect = CGRect(origin: touchPoint, size: touchSize)
-            popoverPresentationController.permittedArrowDirections = .any
-            popoverPresentationController.delegate = self
+        let setupPopover = { [unowned self] in
```

Comment:
why not `[weak self]`?

Reply:
The closure `updateDisplayedPopoverProperties` is only around while BVC is around. So there wont be a case where BVC is nil and the `updateDisplayedPopoverProperties` still exists. So using unowned self here is more convenient